### PR TITLE
Player can move during camera blending between regions now + Code cleanup

### PR DESCRIPTION
**Pull request by jurcaua**
Seems like something we need to playtest for sure, since it may be weird, but I tried to make it as natural as possible.

We keep the current movement axis for movement, and only update it once the player moves in another direction. I noticed for myself that I would try to move relative to the camera always, but if I am going straight through a door or something, then I would want to keep going straight until I changed the direction I'm going.

Updated the Kitchen Mission dentures cutscene to stop player movement as before, and now we stop it right when they enter. Custom logic for that in the mission. Also updated the cutscene camera to better showcase the dentures since the level design changed.

Code cleanup part:
* moved stray scripts that were not in the scripts folder, into the scripts folder
* moved a general function from PlayerController to Utils
* added some more constants for all input axes (didnt know that was the plural of axis https://www.grammar-monster.com/plurals/plural_of_axis.htm ... the more u know)

### CODE REVIEW

File: Assets/Prefabs/Missions/MissionCafeteria1.prefab [hunk 1]
```diff
@@ -43,7 +43,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: afb112b1dbeb55f408e02b5b606efadb, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  respawnPosition: {x: -41, y: 0, z: 37}
+  respawnPosition: {x: -42, y: 0, z: 23}
```
Comment (jurcaua) [inline Assets/Prefabs/Missions/MissionCafeteria1.prefab]: Respawn point BEFORE getting dentures was outside the map so I fix that too.

File: Assets/Scripts/CameraManager.cs [hunk 2]
```diff
@@ -62,7 +62,7 @@ private void EnableZoomOnBlendEnd_Internal(CinemachineVirtualCamera fromCamera,
 
     public ICinemachineCamera GetActiveCamera()
     {
-        return brain.ActiveVirtualCamera;
+        return brain == null ? null : brain.ActiveVirtualCamera as CinemachineVirtualCamera;
```
Comment (chadleytan) [inline Assets/Scripts/CameraManager.cs:65]: brain looooool

**Review by chadleytan — APPROVED** [review]
Good to merge!

File: Assets/Scripts/CameraManager.cs (hunk 2 above)
Comment (jurcaua) [reply Assets/Scripts/CameraManager.cs:65]: LOOOL yee they call it the CinemachineBrain cause it takes over the main camera :))))

**Comment by jurcaua** [comment]
Merci Chad! Gonna wait for people to try it out first before merging!!

**Comment by masonachu** [comment]
feels good to me!

**Pull request merged**